Source/WebCore/rendering/style/StyleSurroundData.h

```diff
@@ -54,6 +54,11 @@ class StyleSurroundData : public RefCounted<StyleSurroundData> {
     bool hasExplicitlySetBorderTopLeftRadius : 1;
     bool hasExplicitlySetBorderTopRightRadius : 1;
 
+    bool hasExplicitlySetPaddingBottom : 1;
```

Comment:
I assume this can't just be `EdgeInsets<bool>` right?

Reply:
This change as-is does not modify size/layout of `StyleSurroundData`.